.goreleaser.yml

```
@@ -55,6 +55,31 @@ builds:
     - goos: windows
       goarch: 386
 
+- id: krane
+  env:
+  - CGO_ENABLED=0
+  main: ./main.go
+  dir: ./cmd/krane
+  binary: krane
+  flags:
+  - -trimpath
+  ldflags:
+  - "-s -w -X github.com/google/go-containerregistry/cmd/crane/cmd.Version={{.Version}}"
+  - "-s -w -X github.com/google/go-containerregistry/pkg/v1/remote/transport.Version={{.Version}}"
```

```suggestion
  - -s
  - -w
  - -X github.com/google/go-containerregistry/cmd/crane/cmd.Version={{.Version}}
  - -X github.com/google/go-containerregistry/pkg/v1/remote/transport.Version={{.Version}}
```

not sure it needs 2 sets of `-s -w`
could either use a list like suggested or squash all into 1 item

---

I just built it with your suggestions and it seems to work. I think we can go on with this.